diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -102,7 +107,7 @@ function StudentQuiz(runtime, element, initData) {
       // shows result of student
       var finalResultHtml = '<div class="html_body">' + result.html_body + '</div>';
       $(finalResult, element).html(finalResultHtml);
-      hideActions();
+//      hideActions();
```

Comment:
@jareerahsan Leftover comment, please remove if this call has become unnecessary.